### PR TITLE
Add missing plugin links to docs page menu

### PR DESCRIPTION
**Pull request by matt852**
# Closes: #3446 
# What's Changed

I updated the documentation webpage to include links to the plugins already uploaded.  The links are merely missing from the menu.

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -230,5 +230,7 @@ nav:
           - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Golden Configuration: "https://docs.nautobot.com/projects/golden-config/"
           - Nautobot Single Source of Truth: "https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
+          - Nautobot Device LifeCycle Management: "https://docs.nautobot.com/projects/device-lifecycle/en/latest/"
+          - Nautobot Device Onboarding: "https://docs.nautobot.com/projects/device-onboarding/en/latest/"
```
Comment (glennmatthews) [inline mkdocs.yml:234]: What's the ordering of entries in this section? Should we impose something?

Comment (matt852) [reply mkdocs.yml:234]: The ordering is exactly as it is listed in this doc.  I personally prefer to alphabetize them, but wasn't sure if a separate PR was warranted to do so.  You good if I do so here?

Screen Shot 2023-03-17 at 12 31 49 PM

Comment (cmsirbu) [reply mkdocs.yml:234]: Original order I think was first-to-get-shiny-new-docs :) Alphabetizing it (based on the second word?) makes a lot of sense.

File: mkdocs.yml [hunk 2]
```diff
@@ -230,5 +230,7 @@ nav:
           - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Golden Configuration: "https://docs.nautobot.com/projects/golden-config/"
           - Nautobot Single Source of Truth: "https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
+          - Nautobot Device LifeCycle Management: "https://docs.nautobot.com/projects/device-lifecycle/en/latest/"
```
Comment (jathanism) [inline mkdocs.yml:233]: Let's go ahead and alphabetize the list while you're touching it. :)

**Review by jathanism — CHANGES_REQUESTED** [review]

**Review by glennmatthews — APPROVED** [review]

**Review by HanlinMiao — APPROVED** [review]

File: mkdocs.yml [hunk 3]
```diff
@@ -226,9 +226,11 @@ nav:
       - Overview: "apps/index.md"
       - Network to Code App Docs:
           - Nautobot ChatOps: "https://docs.nautobot.com/projects/chatops/en/latest/"
+          - Nautobot Device LifeCycle Management: "https://docs.nautobot.com/projects/device-lifecycle/en/latest/"
+          - Nautobot Device Onboarding: "https://docs.nautobot.com/projects/device-onboarding/en/latest/"
           - Nautobot Firewall Models: "https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/"
-          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Golden Configuration: "https://docs.nautobot.com/projects/golden-config/"
+          - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Single Source of Truth: "https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
```
Comment (glennmatthews) [inline mkdocs.yml:234]: Should be working now:

```suggestion
          - Nautobot Single Source of Truth: "https://docs.nautobot.com/projects/ssot/en/latest/"
```

**Pull request merged**